common/util.go

```diff
@@ -26,6 +28,12 @@ import (
 // HTTPTimeout timeout used in HTTP connections between nodes
 var HTTPTimeout = 8 * time.Second
 
+// Max Segment Duration
+var MaxDuration = (5 * time.Minute)
+
+// Max Segment Size in bytes (cap reading HTTP response body at this size)
+var MaxSegSize = int(MaxDuration.Seconds()) * 1024 * 1024 // 1080p30fps video @ 8000Kb/s or 1MiB/s
```

Comment:
This might be obvious, but could you explain this calculation for my own understanding?

Reply:
Ah right brain fart on my end heh 👍

Reply:
Thinking more about this, I'll create a `MaxBitrate` variable here to capture the `1024*1024` think and make it easier on the eyes lol.